bedevere/files.py

```diff
@@ -0,0 +1,20 @@
+"""Checks related to files on a pull request."""
+import gidgethub.routing
+
+from .news import check_news
+from .prtype import classify_by_files
+from . import util
+
+
+router = gidgethub.routing.Router()
+
+
+@router.register('pull_request', action='opened')
+@router.register('pull_request', action='synchronize')
+async def retrieve_files(event, gh, *args, **kwargs):
+    pull_request = event.data['pull_request']
+    filenames = await util.filenames_for_PR(gh, pull_request)
+    await check_news(gh, pull_request, filenames)
+    if event.data['action'] == 'opened':
```

Comment:
Why restrict this to only the initial set of files? What happens if someone gets asked to make a change that expands the scope of the PR?

Reply:
I restricted this because of the ability to add and remove labels manually.  I thought that if someone really wanted to remove a label, then the bot might keep trying to add it back.  I didn't know if it would be better to fix it later if someone complained or just start out by only adding labels to the initial pull.